src/frontend/.gitignore

```diff
@@ -0,0 +1,32 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Can we keep a single the .gitignore in the root folder?

Reply:
Done :D